src/oom/oomd-manager.c

```diff
@@ -232,6 +232,24 @@ static int get_monitored_cgroup_contexts_candidates(Hashmap *monitored_cgroups,
         return 0;
 }
 
+static int update_monitored_cgroup_contexts_candidates(Hashmap *monitored_cgroups, Hashmap **candidates) {
+        _cleanup_hashmap_free_ Hashmap *new_candidates = NULL;
+        int r;
+
+        assert(monitored_cgroups);
```

Comment:
Maybe, it is better to add two more assertions: `assert(candidates); assert(*candidates);`